flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/log/CygnusLogger.java

```diff
@@ -58,7 +80,14 @@ public void error(String componentName, String message) {
      * @param message
      */
     public void fatal(String componentName, String message) {
-        fatal("[" + componentName + "] " + message);
+        try {
+            fatal("[" + componentName + "] " + message);
+        } catch (Exception e) {
+            if (shutdown) {
+                System.err.println("A problem with the logging system was found... shutting down Cygnus right now!");
```

Comment:
Maybe you should also print the Exception trace/information to provide more information to the administrator reading the mesage?